packages/tocco-ui/src/EditableValue/typeEditors/DateRangeEdit.js

```diff
@@ -15,22 +19,27 @@ const DateRangeEdit = props => {
     : null
 
   const handleChange = dates => {
-    const value = dates && dates.length === 2
-      ? {
+    if (dates && dates.length === 2) {
+      props.onChange({
```

Comment:
`onChange` is not a required prop. There should either be a null check here or the prop be declared as required. (it is declared as required in `<DateEdit/>`)